src/systemd.rs

```diff
@@ -228,11 +229,11 @@ fn systemctl(cmd: &str, label: &str, user: bool) -> io::Result<()> {
 }
 
 #[inline]
-fn global_dir_path() -> PathBuf {
+pub fn global_dir_path() -> PathBuf {
```

Comment:
If you want to make public `global_dir_path` and `user_dir_path` from the systemd code, can they either be exposed with a prefix like `systemd_` or within a module like `service_manager::systemd::global_dir_path`?

Otherwise, it isn't obvious for someone accessing those methods that they're specific to systemd.

Reply:
Thanks for the feedback. I amended my commit to add the `systemd` prefixes to the functions.

The amend also contains another little change in the tests. The existence of the user being specified was being asserted before the user account was deleted. I changed the order of that so that the assertion is made *after* the account is deleted. I'm sure I had it that way originally. This ensures the user account for the test will be tidied up even if that assertion fails.